tools/src/FileStatistics.cc

```diff
@@ -55,6 +55,11 @@ void printStatistics(const char *filename) {
         std::cout << "--- Column " << k << " ---" << std::endl;
         std::cout << stripeStats->getColumnStatistics(k)->toString()
                   << std::endl;
+        for(unsigned int r = 0; r < stripeStats->getNumberOfRowIndexStats(k); ++r) {
```

Comment:
Is it better to make printing row index stats optional as it may be too verbose?